.github/workflows/tf-apply.yaml

```diff
@@ -41,7 +41,7 @@ permissions:
 jobs:
   apply:
     name: Terraform Apply ${{ inputs.branch }}
-    runs-on: X64
```

Comment:
Couldnt get it to run on X64, was there a reason for this runner?

Reply:
The amount of time it could take, I'd suggest making it an input defaulted to X64 and pass down ubuntu latest from the caller rather than swapping one for the other.